### ISSUE TITLE
Minimize, Maximize, and Close Buttons Are Non-Functional on Windows

### ISSUE THREAD

**Issue by Enoq00** [ekvedaras/redis-gui#95]
Description: The Minimize, Maximize/Restore, and Close buttons in the title bar (top-right corner of the window) are unresponsive on the Windows version of the application.

Steps to Reproduce:
Open the application on a Windows machine.
Try clicking the "Minimize" (-), "Maximize/Restore" (☐), or "Close" (X) buttons on the top-right corner of the window.
Expected Behavior:
The buttons should function as follows:

"Minimize" should minimize the application to the taskbar.
"Maximize/Restore" should toggle between fullscreen and windowed mode.
"Close" should close the application.
Observed Behavior:
None of these buttons respond to user interaction.

Environment:
OS Name:                       Microsoft Windows 11 Pro
OS Version:                    10.0.26100 N/A Build 26100
App Version: Latest at the moment (Didn't found it on application)
Please address this issue as it significantly impacts usability. Let me know if more details or logs are needed.

**Comment by Enoq00** [comment]
double click on header minimize works well.

**Comment by ekvedaras** [comment]
Yes, aware of this one. Related to underlying package for title bars. No idea what happens there. Don't have Windows machine to fix either. Happy for you to take a stab 🙂 Maybe there's a new update in that package that addresses it or maybe there are better packages 🤷
